libs/filamat/src/Includes.cpp

```diff
@@ -261,10 +261,17 @@ std::vector<FoundInclude> parseForIncludes(const utils::CString& source) {
         const size_t nameStart = result;
 
         // Increment until we reach the next "
-        while (sourceString[result] != '"') {
+        while (result < sourceString.length() && sourceString[result] != '"') {
             result++;
         }
 
+        // check we're not at the end of the line -- this would be a malformed include directive.
+        if (result >= sourceString.length()) {
+            result = sourceString.find("#include", result);
```

Comment:
Do we need the call to `.find` here?

Reply:
I have no idea, I just copied how that code handles EOF above. It looks like it tries to ignore that include and find the next one?